src/components/Input/Input.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styles from './Input.module.css';
+
+const Input = ({
```

Comment:
I would also recommend adding docs to the component. 
Since we don’t use TypeScript, the IDE won’t suggest possible props, so docs will make working with the component much easier.

Here's the example:
![CleanShot 2025-04-30 at 00 21 16@2x](https://github.com/user-attachments/assets/1dcd1d1f-8e99-41dc-8a0e-24442701c79b)